lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -877,6 +880,42 @@ def write(chunk)
       end
     end
 
+    def dynamic_target_index_enabled?()
+      @dynamic_target_index && @logstash_format && @id_key && (@write_operation == UPDATE_OP || @write_operation == UPSERT_OP)
+    end
+
+    def get_dynamic_target_indices(chunk)
```

Comment:
`get_` prefix is redundant.
And `dynamic_target_indices` is a bit of ambiguous for me.

Reply:
ok, will use some other name e.g. affinity_indices?